### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by ShreyasMandlik**
## What

Please include a summary of the changes.

- [x] Added skeleton for backend.
- [x] Created github actions for eslint and test 

# Checklist

- [ ] My code follows the style guidelines of this project and is properly formatted
- [ ] I have performed a self-review of my own changes
- [ ] All existing tests pass, and I have added new ones if required

### CODE REVIEW

**Review by Charvit123 — APPROVED** [review]
Looks good!!

File: .github/workflows/eslint.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./backend
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14"
+      - run: npm i eslint @types/eslint
```
Comment (utsav00) [inline .github/workflows/eslint.yml:15]: No need to install specific packages here, just run `npm i` here

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    defaults:
+      run:
+        working-directory: ./backend
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14"
+      - run: npm i @nestjs/testing 
+      - run: npm i jest
```
Comment (utsav00) [inline .github/workflows/test.yml:17]: Same as above, can just use `npm i` here

File: .github/workflows/eslint.yml (hunk 1 above)
Comment (ShreyasMandlik) [reply .github/workflows/eslint.yml]: Resolved

File: .github/workflows/test.yml (hunk 2 above)
Comment (ShreyasMandlik) [reply .github/workflows/test.yml]: Resolved

**Pull request merged**